_episodes/01-getting-started-with-conda.md

```diff
@@ -198,7 +202,8 @@ your usual version of Python in your normal environment.
 
 ## Why use Conda?
 
-Whilst there are many different package and environment management systems that solve either the
+In Python there is a built in package manager `pip` and virtual environment manager `venv`, so
+why use Conda? Whilst there are many different package and environment management systems that solve either the
 package management problem or the environment management problem, Conda solves both of these
 problems and explicitly targeted at (data) science use cases.
```

Comment:
Not your edit but needs an `is` in here or changing `targeted` to `targets`...

```suggestion
problems and is explicitly targeted at (data) science use cases.
```

...org...

```suggestion
problems and explicitly targets (data) science use cases.
```